Docs: Improve remote image renderer documentation (#20031)
grafana/grafana: docs/sources/administration/image_rendering.md

The [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can also be run as a remote HTTP rendering service. In this setup Grafana will render an image by making a HTTP request to the remote rendering service, which in turn render the image and returns it back in the HTTP response to Grafana.

> As an alternative to installing and running the image renderer as a plugin you can run it as a remote image rendering service using Docker.

Reviewer: Think you can rewrite this to describe that the remote HTTP rendering service can either be run as a node.js app (from source) or in a docker container.

> For running grafana with remote image renderer create a `docker-compose.yml` with the following content.

Reviewer: Just suggestion, but not sure if it's a good one. Please change as you see fit.

```suggestion
**Using Docker:**
The following example describes how to run Grafana and the remote HTTP rendering service in two separate docker containers using Docker Compose.  

Create a `docker-compose.yml` with the following content.
```

The [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can also be run as a remote HTTP rendering service. In this setup Grafana will render an image by making a HTTP request to the remote rendering service, which in turn render the image and returns it back in the HTTP response to Grafana.

> As an alternative to installing and running the image renderer as a plugin you can run it as a remote image rendering service running as a node.js application or inside a Docker container.

Reviewer: ```suggestion
You can run the remote HTTP rendering service using Docker or as a standalone Node.js application.
```

> **Running as standalone node.js application:**

Reviewer: I would move `Using Docker` section above this section since Docker should be the preferred/first thing to use when considering remote HTTP rendering service.
